hubblestack/hec/obj.py

```diff
@@ -131,6 +131,7 @@ def __str__(self):
 
 
     def __init__(self, token, http_event_server, host='', http_event_port='8088',
+                 http_event_index='default',
```

Comment:
Let's move http_event_index next to token and don't give it a `default`. It should always be defined.